README.md

```diff
@@ -1,45 +1,20 @@
-# Introduction
+# Kafka Connect Datadog Logs
 
-This project and README are WIP!
+kafka-connect-datadog-logs is a [Kafka Connector](http://kafka.apache.org/documentation.html#connect) for sending 
+records from Kafka to the Datadog Event Intake API.
 
-# Running in development
+# Development
 
+You can build kafka-connect-datadog-logs with Maven using the standard lifecycle phases.  To build the connector run 
+`mvn clean compile package`, which will create a jar file in the `target` directory that you can use with Kafka Connect.
 
-The [docker-compose.yml](docker-compose.yml) that is included in this repository is based on the Confluent Platform Docker
-images. Take a look at the [quickstart](http://docs.confluent.io/current/cp-docker-images/docs/quickstart.html#getting-started-with-docker-client)
-for the Docker images.
+The jar file for use on [Confluent Hub](https://www.confluent.io/hub/) can be found in `target/components/packages`.
```

Comment:
Is it possible to use `kafka-connect-datadog-logs` outside Confluent Hub? If yes, it would be great to explain how to do it.

Reply:
Yes it is, through maven. I will do that.